docs/list.md

```diff
@@ -424,6 +424,14 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
 
 ### O
 
+#### *Odigos*
+
+- **Swag**: Swag (Specific Swags not mentioned)
```

Comment:
The swag is mentioned in the[ Readme file](https://github.com/keyval-dev/odigos/blob/main/README.md), can you please update this?